easybuild/easyconfigs/c/canu/canu-2.3-GCCcore-13.3.0-Java-11.eb

```
@@ -23,10 +23,16 @@ dependencies = [
     ('Java', '11', '', SYSTEM),
     ('Perl', '5.38.2'),
     ('gnuplot', '6.0.1'),
+    # mark next line if the environment has OpenSSL3
+    ('OpenSSL', '3', '', SYSTEM),
 ]
 
 start_dir = 'src'
 
+
+build_cmd = r'sed -i "s|java = .*command.*;|'
+build_cmd += 'java = \'${EBROOTJAVA}/bin/java\';|" pipelines/canu/Defaults.pm && make'
```

better to use prebuildopts for this:

```suggestion
prebuildopts = r'sed -i "s|java = .*command.*;|'
prebuildopts += 'java = \'${EBROOTJAVA}/bin/java\';|" pipelines/canu/Defaults.pm && '
```